html/templates/header.html

```diff
@@ -5,6 +5,7 @@
   <link rel="stylesheet" type="text/css" href="changelog.css">
   <base target="_blank" />
   <script type='text/javascript'>
+  var DISCORDURL = CONFIG_GET(string/DISCORDURL)
```

Comment:
How tf does this work

Reply:
that isn't meant to be in there, didn't mean to merge it